api/DOMTokenList.json

```diff
@@ -833,10 +833,10 @@
               "notes": "Before Chrome 50, this property was part of the deprecated child <code>DOMSettableTokenList</code> interface."
             },
             "edge": {
-              "version_added": "≤18"
+              "version_added": "17"
```

Comment:
Trusting this change. Just a note for anyone following along that `value` is not `values`, it's not implied by `iterable<T>` in IDL. It's not strange that it was added separately.